### DISCUSSION TITLE
Welcome to openbullet Discussions!

### DISCUSSION THREAD

**Discussion by openbullet** [openbullet/openbullet#781] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by verticalsync** [comment]
why

**Reply by openbullet** [reply]
why not?

**Reply by verticalsync** [reply]
idk brother

**Comment by ZDD19980506** [comment]
May I have your contact information

**Reply by openbullet** [reply]
`ruri (at) openbullet [dot] dev`
